keyboards/contra/keymaps/ryanm101/keymap.c

```diff
@@ -47,208 +46,208 @@ enum {
+   * |------+------+------+------+------+-------------+------+------+------+------+------|
+   * |      |  F5  |  F6  |  F7  |  F8  |      |      |   4  |   5  |   6  |  /   |      |
+   * |------+------+------+------+------+------|------+------+------+------+------+------|
+   * |      |  F1  |  F2  |  F3  |  F4  |      |      |   1  |   2  |   3  |  +   |KP_ENT|
+   * |------+------+------+------+------+------+------+------+------+------+------+------|
+   * | Ctrl | GUI  | Alt  |      |      |      |      |      |   0  |   .  |  -   |      |
+   * `-----------------------------------------------------------------------------------'
+   */
+  [_L2] = LAYOUT_ortho_4x12(
+    KC_SYSREQ, KC_F9,   KC_F10,  KC_F11,  KC_F12,  XXXXXXX, KC_NUMLOCK, KC_KP_7, KC_KP_8, KC_KP_9,   KC_KP_ASTERISK, KC_BSPC,
+    XXXXXXX,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   XXXXXXX, XXXXXXX,    KC_KP_4, KC_KP_5, KC_KP_6,   KC_KP_SLASH,    _______,
+    XXXXXXX,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   XXXXXXX, XXXXXXX,    KC_KP_1, KC_KP_2, KC_KP_3,   KC_KP_PLUS,     KC_KP_ENTER,
+    _______,   _______, _______, XXXXXXX, XXXXXXX, XXXXXXX, KC_SPC,     XXXXXXX, KC_KP_0, KC_KP_DOT, KC_KP_MINUS,    _______
+  ),
+
+  /* L3
+   * ,-----------------------------------------------------------------------------------.
+   * |RGBTOG|RGBHUI|RGBHUD|RGBSAI|RGBSAD|      |      | PGDN | PGUP |      |      |      |
+   * |------+------+------+------+------+-------------+------+------+------+------+------|
+   * |BLTOG |BLSTEP|RGBVAI|RGBVAD|RGBMOD|RGBRMD| Left | Down |  Up  | Right|      |      |
+   * |------+------+------+------+------+------|------+------+------+------+------+------|
+   * | TMUX |      |      |      |      |      |      |      |      |      |      |Enter |
+   * |------+------+------+------+------+------+------+------+------+------+------+------|
+   * |      |      |      |      |      |      |      |      |      |      |      |      |
+   * `-----------------------------------------------------------------------------------'
+   */
+  [_L3] = LAYOUT_ortho_4x12(
+    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_PGDN, KC_PGUP, XXXXXXX, XXXXXXX, XXXXXXX,
+    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT, XXXXXXX, XXXXXXX,
+    TMUX,    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_ENT,
+    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, KC_SPC,  XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX
+  )
+
+;
 
 // Parantheses
-void paranthesis_dance (qk_tap_dance_state_t *state, void *user_data) {
-  if (state->count == 1) {
+void paranthesis_dance (qk_tap_dance_state_t *state, void *user_data)
+  if (state->count == 1)
     SEND_STRING("()"); register_code(KC_LEFT); unregister_code(KC_LEFT);
-  } else if (state->count == 2) {
+   else if (state->count == 2)
     SEND_STRING("(");
-  } else if (state->count == 3) {
```

Comment:
Curly bracket got edited out. Needs to be there

Reply:
Definitely was trying to remove in a selection and had the option disabled.